common/spring-cloud-stream-modules-common-configuration/src/test/java/org/springframework/cloud/stream/module/ftp/FtpSessionFactoryPropertiesTests.java

```diff
@@ -98,6 +99,7 @@ public void noPropertiesThrowsMeaningfulException() {
 			context.register(Conf.class);
 			context.refresh();
 			fail("should throw exception");
```

Comment:
Yes, I am not at all a fan of `@Test(expected=` - I prefer to do further evaluation of the exception to verify it's the exact one I wanted.